pkg/config/analysis/msg/messages.yaml

```diff
@@ -50,16 +50,16 @@ messages:
       - name: podName
         type: string
 
-  - name: "GatewayPortNotOnWorkload"
+  - name: "GatewayPortNotDefinedOnService"
```

Comment:
I'm a bit concerned about whether we should replace the message with a new one or overwrite it. Since users may already understand the semantics of IST0104, they might be confused as to why they no longer see the old message and now have a new one. I think a good practice would be to generate a new message, `GatewayPortNotDefinedOnService`, and retire the old one (like other `RETIRED` messages). However, I'm not sure if this approach aligns with what others have done in the past though.